source/tutorials/explore-change-streams.txt

```diff
@@ -0,0 +1,230 @@
+.. _kafka-tutorial-explore-change-streams:
+
+==============================
+Explore MongoDB Change Streams
+==============================
+
+Follow this tutorial to learn how to create a change stream on a MongoDB
+collection and observe the change events it creates.
+
+If you already understand how to create change streams and the change
+event data they create, proceed to the
+(TODO: link ``Sourcing Data from MongoDB``) tutorial.
+
+Before you start this tutorial, you must complete the steps in the
+Tutorial Setup (TODO: link).
+
+.. procedure::
+   :style: connected
+
+   .. step:: Connect to the Docker Container
+
+      Create two interactive shell sessions on the tutorial Docker
+      Container, each in a separate window.
+
+      .. code-block:: bash
+         :copyable: true
+         :caption: This command starts an interactive shell called shell1
+
+         docker run --rm --name shell1 --network mongodb-kafka-base_localnet -it robwma/mongokafkatutorial:latest bash
```

Comment:
Issue I ran into:  Ran into the following error when testing the provided environment. Followed the rest of the tutorial locally and it worked well.

```
docker run --rm --name shell1 --network mongodb-kafka-base_localnet -it robwma/mongokafkatutorial:latest bash
WARNING: The requested image's platform (linux/arm64/v8) does not match the detected host platform (linux/amd64) and no specific platform was requested
docker: Error response from daemon: network mongodb-kafka-base_localnet not found.
```

Reply:
No, the warning is not related to that as it runs ok in emulation. I tested on an intel MBP.

My container is still running from when I tested, but I'll try it again. I'll try to look into the error you're getting.

```
9f8792f7a307   robwma/mongokafkatutorial:latest            "bash"                   3 days ago   Up 3 days                                                                shell1
```